README.md

```diff
@@ -68,14 +68,12 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 |---|---|
 |**Read** access to code & metadata | To sync development information to Jira|
 |**Read** and **write** access to issues and pull requests| To power Smart Commit actions and unfurl Jira URLs|
+|**Read** and **write** access to actions and deployments| To sync CI/CD (GitHub Actions) information to Jira|
 
 ## Using the integration
 
 ### Development information in Jira
-To access the development information in Jira...
-
-### Take action using Smart Commits
-Smart Commits are documented on the [Atlassian site](https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html) and allow you to comment on Jira issue in commit messages, branches, and pull requests. For example: `[JRA-123] fix typo` will be sent through to Jira and appear in the Development Information section of the Jira issue with the key `JRA-123`
+To start seeing your development information from GitHub in Jira, you'll need to start using Smart Commits. Smart Commits are documented on the [Atlassian site](https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html) but you'll have to simply make sure a Jira issue is a part of the commit message. For example: `[JRA-123] fix typo` will be sent through to Jira and appear in the Development Information section of the Jira issue with the key `JRA-123`. Any branch, commit, pull request, build and deployment linked to this commit will now appear in Jira. 
```

Comment:
No, they don't need to use Smart Commits. Smart Commits are for changing the state of an issue, like logging time or adding a comment. Smart Commits are optional!

Instead we should probably link to this page: https://support.atlassian.com/jira-software-cloud/docs/reference-issues-in-your-development-work/. And make clear that all they need to do is to add a Jira issue key to their commit message, branch name, or PR title.